test/Main.test.ts

```diff
@@ -1715,6 +1725,357 @@ describe(`MainP${IMPLEMENTATION} contract`, () => {
         .withArgs([token0.address], [fp('1')], [ethers.utils.formatBytes32String('USD')])
     })
 
+    describe.only('Historical Redemptions', () => {
```

Comment:
remove only

Reply:
Yup; I hadn't made a pass on the tests yet for the record